net-mgmt/pfSense-pkg-softflowd/files/usr/local/pkg/softflowd.inc

```diff
@@ -159,4 +161,29 @@ function cleanup_config_softflowd() {
 	$a_pkg[0] = $pffconf;
 }
 
+function display_interface_indices_softflowd() {
+	global $config;
+
+	if (is_array($config['installedpackages']['softflowd']['config'][0])) {
+		$cf = $config['installedpackages']['softflowd']['config'][0];
+	} else {
+		$cf = array();
+	}
+	
+	/* Compile and display the list of interface indices if softflowd is active */
+	if (isset($cf) && isset($cf['interface'])) {
```

Comment:
isset($cf) will be always true here, so it can be removed. Only $cf['interface'] must be checked